topology/probes/ovsdb/ovsdb.go

```diff
@@ -85,13 +85,14 @@ func (o *OvsdbProbe) OnOvsBridgeAdd(monitor *ovsdb.OvsMonitor, uuid string, row
 	defer o.Unlock()
 
 	name := row.New.Fields["name"].(string)
+        datapath_type := row.New.Fields["datapath_type"].(string)
```

Comment:
can you use  columnStringValue it is safer

Reply:
nit: wrong indentation